pkgs/top-level/all-packages.nix

```diff
@@ -12301,7 +12301,7 @@ with pkgs;
 
   sanctity = callPackage ../tools/misc/sanctity { };
 
-  sasquatch = callPackage ../tools/filesystems/sasquatch { };
+  sasquatch-v4be = callPackage ../by-name/sa/sasquatch/package.nix { bigEndian = true; };
```

Comment:
@vlaci could you please maybe move this into `by-name/sa/sasquatch-v4be` as well ?

Reply:
Is this how you'd like it?
https://github.com/NixOS/nixpkgs/pull/219169/files#diff-276d44079aa310eda1d4e6bc92d2ffedcf4c0cf9fb5c5f988502809efc60089fR3

Reply:
Yep thanks a lot for your work on this !